### PR TITLE
line detection setup for stereo camera in gazebo

### PR DESCRIPTION
**Pull request by bsubei**
rosparams set the topics in launch files.
solved lag issue by playing with buff_size.

### CODE REVIEW

**Comment by chicagoedt** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.chicagoedt.org/job/Software_IGVC_Upstream/41/
Test Passed.

**Comment by chicagoedt** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.chicagoedt.org/job/Software_IGVC_Upstream/42/
Test Passed.

File: line_detection/launch/stereo_gabor.launch [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<launch>
+
+  <node pkg="line_detection" name="gabor_linedetection" type="gabor-linedetection.py" ns="left" output="screen">
+    
+    <param name="use_mono" type="bool" value="true" />
+    <param name="use_compressed_format" type="bool" value="false" />
+    <param name="subscriber_image_topic" type="str" value="stereo_camera/left/image_raw" />
```
Comment (l0g1x) [inline line_detection/launch/stereo_gabor.launch]: this should be divided into a camera namespace parameter as well, incase we case the camera we are using (e.g. bumblebee or duo)

Comment (bsubei) [inline line_detection/launch/stereo_gabor.launch]: so the namespace parameter should be separate?

**Comment by chicagoedt** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.chicagoedt.org/job/Software_IGVC_Upstream/43/
Test Passed.

**Pull request merged**